scripts/sct_compute_mtsat.py

```diff
@@ -199,20 +199,11 @@ def main(argv):
                                          nii_b1map=nii_b1map)
 
     # Output MTsat and T1 maps
-    # by default, output in the same directory as the input images
     sct.printv('Generate output files...', verbose)
-    if args.omtsat is None:
-        fname_mtsat = os.path.join(os.path.dirname(nii_mt.absolutepath), "mtsat.nii.gz")
-    else:
-        fname_mtsat = args.omtsat
-    nii_mtsat.save(fname_mtsat)
-    if args.ot1map is None:
-        fname_t1map = os.path.join(os.path.dirname(nii_mt.absolutepath), "t1map.nii.gz")
-    else:
-        fname_t1map = args.ot1map
-    nii_t1map.save(fname_t1map)
+    nii_mtsat.save(args.omtsat)
+    nii_t1map.save(args.ot1map)
```

Comment:
A consideration: ~15 of our scripts are currently using the  `-ofolder` argument to accompish a similar thing, e.g.

https://github.com/neuropoly/spinalcordtoolbox/blob/432bbc681d8f7fdb27d007f0f782eaf0285582be/scripts/sct_fmri_moco.py#L93-L99

https://github.com/neuropoly/spinalcordtoolbox/blob/432bbc681d8f7fdb27d007f0f782eaf0285582be/scripts/sct_propseg.py#L199-L204

File output locations in CLI scripts could be a larger discussion in a different issue, though. (The locations are not always consistent, for example if you compare `sct_testing_data/` before/after running `sct_testing`.)

Reply:
> if you compare sct_testing_data/ before/after running sct_testing, files are output in a few different places.

Yes I noticed this yesterday and created #2911. In my opinion:
- Unit tests should not leave artifacts on the filesystem. The output should be a PASS/FAIL + traceback, etc only.
- If a script generates a lot of output data, it's perfectly reasonable to have an `ofolder` parameter. If it only generates a file or two, specifying the output file paths is preferable.

Reply:
I think that's a very reasonable take. I'm going to update the Wiki with this information... I think the Testing page needs a bit of an update. :)